__tests__/__integration__/ZoweNode.integration.test.ts

```diff
@@ -16,9 +16,9 @@ import * as chaiAsPromised from "chai-as-promised";
 // tslint:disable-next-line:no-implicit-dependencies
 import * as expect from "expect";
 import * as vscode from "vscode";
-import { ZoweDatasetNode } from "../../src/ZoweDatasetNode";
+import { ZoweDatasetNode } from "../../src/dataset/ZoweDatasetNode";
```

Comment:
I have gone through the test files to see which functions are covered right now in tests...this file is only testing zoweDatasetNode functions as far as I can tell...so that would make sense to me :)

In my opinion the test file structure should exactly match the ts file structure. So you would have a test file for each ts file. But this is more referring to unit tests than integration tests...right now we have enough unit tests to implement the same structure for unit tests, but for integration tests I think we would have a lot of empty files...